analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -577,3 +580,47 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
         }
     }
 }
+
+/**
+ * Several Maven repositories have disabled HTTP access and require HTTPS now. To be able to still analyze old Maven
+ * projects that use the HTTP URLs, this [MirrorSelector] implementation automatically creates an HTTPS mirror if a
+ * [RemoteRepository] uses a disabled HTTP URL. Without that Maven would abort with an exception as soon as it tries to
+ * download an Artifact from any of those repositories.
+ *
+ * **See also:**
+ *
+ * [GitHub Security Lab issue](https://github.com/github/security-lab/issues/21)
+ * [Medium article](https://medium.com/@jonathan.leitschuh/update-want-to-take-over-the-java-ecosystem-all-you-need-is-a-mitm-d069d253fe23)
```

Comment:
You could use the short link `https://medium.com/p/d069d253fe23` so you don't have to `@Suppress("MaxLineLength")`.

Reply:
Thanks, done.